config/config.go

```diff
@@ -195,6 +195,23 @@ func (sType ServerType) String() string {
 	return "Unknown"
 }
 
+func EnabledServers() []string {
```

Comment:
@haoming29 Is there any reason that you have enabledServers as private?

Reply:
I think the intent was to enforce us to use getters and setters to change the value of the variable instead of directly assign value to it (it's a int type under the hood)